### DISCUSSION TITLE
What to do after filling the credentials in the <username>.md file?

### DISCUSSION THREAD

**Discussion by shubh1007** [community/community#17782] [GitHub Education]
I have forked the repository and then created the repository inside _data folder. 
After that created the <username>.md file. 
There I have filled my credentials. 
Now, what to do?

**Comment by aryatalathi** [comment]
Create a pull request from the Pull Requests tab. Go through the checklist on the pull request template to guarantee your submission is valid. Create a request, the GitHub Education team will review your application.
